src/modules/Builder/features/ActivityItems/ItemConfiguration/InputTypeItems/PhrasalTemplate/PhrasalTemplateField/PhrasalTemplateRenderField.tsx

```diff
@@ -78,10 +83,36 @@ export function RenderedField({
       return missedItem?.name;
     }
 
-    if (selectedItem?.responseType !== displayMode?.id) {
+    if (
+      selectedItem?.responseType !== displayMode?.id &&
+      selectedItem?.responseType !== ItemResponseType.SliderRows
+    ) {
+      const items = DisplayModeOptions[selectedItem?.responseType || 'default'];
```

Comment:
`default` isn't an acceptable property of the `DisplayModeOptions` object – that key doesn't exist. TypeScript is currently allowing this code to pass, presumably due to the extra expression complexity, but you can see that `default` is actually invalid if you were to try to access it directly:

![CleanShot 2024-08-12 at 15 24 42@2x](https://github.com/user-attachments/assets/13368735-f6c9-440a-ab7e-55870c623aa8)

Instead, I would do this:

```suggestion
      const items = selectedItem ? DisplayModeOptions[selectedItem.responseType] : undefined;
```

Reply:
hmm I forgot to add the default, is better to add the default because it's acting as a switch statement and the idea is stop if-ting everything. I'll add the default as null as the other keywords